test/org/pentaho/di/trans/steps/pentahoreporting/PDI7502Test.java

```diff
@@ -0,0 +1,176 @@
+/*! ******************************************************************************
```

Comment:
This is an integration test IMO - please move this over and rename with IT in the name. Thanks!